tests/governance.py

```diff
@@ -654,7 +813,11 @@ def gov(args):
         test_all_nodes_cert_renewal(network, args)
         test_service_cert_renewal(network, args)
         test_service_cert_renewal_extended(network, args)
-        test_cose_auth(network, args)
+        if args.authenticate_session != "COSE":
```

Comment:
```suggestion
        if args.authenticate_session == "COSE":
```

Unless I'm missing something...

Reply:
Ah ha! Ta. That clears up the comment below as well. So I think the current tests are:

`sessions_auth`
- client-authenticated TLS
- using HTTP signatures for governance,
- with additional tests that COSE signatures are accepted
`sessions_noauth`
- anonymous TLS
- using HTTP signatures for governance
- with additional tests that COSE signatures are accepted
`sessions_coseauth`
- client-authenticated TLS (?)
- using COSE signatures for governance
- skipping additional COSE tests

That's good coverage for now, but we should refactor to simplify this at some point.

Reply:
I had this same question, thanks for clearing it up